www_src/lib/validators.js

```diff
@@ -0,0 +1,98 @@
+module.exports = {
+
+  validators: {
+    email: {
+      regex: /^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/i,
+      message: 'Please use a valid email address.'
+    },
+    passwordLength: {
+      regex: /^\S{8,128}$/,
+      message: 'Must be between 8 and 128 characters.'
+    },
+    lowerCase: {
+      regex: /[a-z]+/,
+      message: 'Must contain a lower case letter.'
+    },
+    upperCase: {
+      regex: /[A-Z]+/,
+      message: 'Must contain an upper case letter.'
+    },
+    numbers: {
+      regex: /\d+/,
```

Comment:
Rules are here (id.webmaker.org uses the default settings) https://github.com/cadecairos/PassTest